agent/chaldea/__init__.py

```
@@ -15,11 +16,138 @@
+def _cache_path(name: str) -> str:
+    return os.path.join(CACHE_DIR, f"{name}.json")
+
+
+def _load_cache_file(path: str) -> Optional[dict]:
+    if not os.path.isfile(path):
+        return None
+    try:
+        with open(path, "r", encoding="utf-8") as f:
+            data = json.load(f)
+        mfaalog.info(f"[Chaldea] 命中本地缓存: {path}")
+        return data
```

Ruff (0.16.2)

[warning] Consider moving this statement to an `else` block

(TRY300)